database/seeders/DummyDataSeeder.php

```diff
@@ -26,6 +27,10 @@ public function run(): void
         $organizations = Organization::factory(10)->create();
         $speakers = Speaker::factory(25)->create();
 
+        News::factory(25)->create([
+            "user_id" => $user,
+        ]);
+
```

Comment:
```suggestion
        News::factory(25)->for($user)->create();
```
Try do this like that.